api/v1alpha1/jobset_webhook_test.go

```diff
@@ -23,6 +24,10 @@ var TestPodTemplate = corev1.PodTemplateSpec{
 }
 
 func TestJobSetDefaulting(t *testing.T) {
```

Comment:
Should you have a test case with `OperatorAny` and label selector set?

Reply:
Good idea - added.